### DISCUSSION TITLE
Accidentally deleted user store, but paret0.com is okay!

### DISCUSSION THREAD

**Discussion by mikhael28** [mikhael28/paretOS#120] [Announcements]
Hey everyone, a few days ago I accidentally deleted a Cloudformation stack that had out user information on it! I've gone ahead and rebuilt that stack and redeployed to the `main` branch the updated .env variables for the auth - so, we can now log in/sign up on paret0.com!

Unfortunately, this means that all the contributing team will have to make new usernames/passwords. I will make sure to add some safeguards, to make sure this doesn't happen again. Sorry all for the hassle, will have feedback on PR's soon, thanks for patience. I will also add this repository to Hacktoberfest, as I have had @LeoUCon and @nikhil12377 recommended that to me!

**Comment by akashshyamdev** [comment]
@mikhael28 good to know 👍 

Do PRs which have been merged before October count in hacktober fest(I had contributed 2-3 PRs so I'm asking)?

**Reply by mikhael28** [reply]
I don't know much about that, going to be doing research about that in the next day or so.
